plugins/bazaar/src/components/HomePageBazaarInfoCard/HomePageBazaarInfoCard.tsx

```diff
@@ -22,6 +22,7 @@ import {
   IconButton,
   makeStyles,
 } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
```

Comment:
Any particular reason not to import `Typography` on line 18?

Reply:
Same as previous comment, MUI recommends full path import, probably to offload tree shaking, can't remember the exact reason